### PR TITLE
Upgrade section for nxrm3.0

### PR DESCRIPTION
**Pull request by dulanism**
JIRA: https://issues.sonatype.org/browse/NEXUS-10038
links out the KB article, as an example

### CODE REVIEW

**Comment by peterlynch** [comment]
not against the deploy-locally.sh file mode change but should not be part of this PR

File: chapter-installation.asciidoc [hunk 1]
```diff
@@ -196,6 +196,57 @@ directory'. More details can be found in <<directories>>.
...
+distribution file. To keep the upgrade simple be sure to schedule some downtime.
+
+NOTE: Upgrading to {oss} 3.0.0 can only be performed by users who run the milestone 7 release of the repository 
+manager. Be sure to manually back up the milestone 7 data dictionary to another location. It is a crucial step to 
+properly upgrade the application.
+
+==== Upgrade with the Installer Application
+
+To upgrade using the installer, locate the data dictionary of your previous installation, then stop the 
+repository manager. Start the installer after downloading. It will automatically detect the existing install,
```
Comment (peterlynch) [inline chapter-installation.asciidoc]: there are some subtleties missing here. The aim should be to minimize downtime. the instructions as ordered here ask the user to stop Nexus before the've even downloaded/started the new installer.

File: chapter-installation.asciidoc [hunk 2]
```diff
@@ -196,6 +196,57 @@ directory'. More details can be found in <<directories>>.
...
+Click 'Next>' to get the screen that displays a field for 'Data Dictionary'. Enter a path to the previous
+data dictionary directory. Inside the target directory, create an empty file called +clean_cache+. Click 'Next>'.
+
+The last screen of the installer will prompt you before it overwrites any files. Choose 'Yes' or 'Yes to all' to 
+overwrite existing files.
+
+==== Upgrade with a Distribution File
+
+To upgrade using a +tar.gz+ or +zip+ file, stop the repository manager, if running. Then download the 
+distribution file applicable to your operating system. Extract the appropriate file.
```
Comment (peterlynch) [inline chapter-installation.asciidoc]: minimize downtime

**Comment by peterlynch** [comment]
for now I would be content with a link to the article and summarizing what can and cannot be upgraded. I see no point in just half duplicating the article

We could expand this book section when we want an upgrade section for 3.0.0 to 3.0.1 or 3.1.

**Comment by dulanism** [comment]
Thanks for your feedback @peterlynch. Though i thought this might be the rare instance where adapting the KB article for a critical process such as updating/upgrading would work, and similar to v2's upgrading section. Will summarize to meet your criteria.

**Comment by mosabua** [comment]
Fine by me either way.. +1 ;-)

**Comment by dulanism** [comment]
Summarized and linked out to KB, per @peterlynch's review

**Pull request merged**